x/pocketcore/types/session.go

```diff
@@ -255,7 +256,8 @@ func (sh SessionHeader) HashString() string {
 func (sh SessionHeader) Bytes() []byte {
 	res, err := json.Marshal(sh)
 	if err != nil {
-		panic(fmt.Sprintf("an error occured converting the session header into bytes:\n%v", err))
+		fmt.Println(fmt.Errorf("an error occured converting the session header into bytes:\n%v", err))
+		os.Exit(1)
```

Comment:
```suggestion
  log.Fatalf("an error occured converting the session header into bytes:\n%v", err)
```
stdlib package `log` offers `log.Fatalf` which logs the error with a timestamp & calls onto os.Exit(1)